server.js

```diff
@@ -4,23 +4,42 @@ require('dotenv').config();
 const {configuration, OpenAI } = require ('openai');
 const app = express();
 
-// const openai = new OpenAI({
-//     apiKey: process.env.openaiAPI,
+const openai = new OpenAI({
+    apiKey: process.env.openaiAPI,
+});
```

Comment:
Commented this out but this is very similar to the active code. It includes a catch error.